hazelcast/src/test/java/com/hazelcast/jet/impl/submitjob/clientside/upload/JobUploadClientFailureTest.java

```diff
@@ -64,6 +65,7 @@
 public class JobUploadClientFailureTest extends JetTestSupport {
 
     private static final String SIMPLE_JAR = "simplejob-1.0.0.jar";
+    private static final String PARALLEL_JAR = "paralleljob-1.0.0.jar";
```

Comment:
Where we can find the source of these jars?

Reply:
I have created locally . But the code is given in the description of the GH request : https://github.com/hazelcast/hazelcast/issues/24143